source/jsonnet/england-wales/individual/blocks/identity-and-health/arrive_in_country.jsonnet

```diff
@@ -90,25 +90,7 @@ function(region_code, census_month_year_date) {
     },
     {
       goto: {
-        block: 'national-identity',
-        when: [
-          {
-            id: 'arrive-in-country-answer',
-            condition: 'less than',
-            date_comparison: {
-              value: census_month_year_date,
-              offset_by: {
-                years: -1,
-              },
-            },
-          },
-          rules.under3,
-        ],
-      },
-    },
-    {
-      goto: {
-        block: if region_code == 'GB-WLS' then 'understand-welsh' else 'language',
+        block: 'past-usual-household-address',
```

Comment:
Isn't under 1 meant to be handled in this blocks routes too?

Reply:
Under 1 goes to length-of-stay and then straight to national-identity.

Reply:
But there's no when condition within arrive_in_country to handle that?

Reply:
there is, if you are under 1 then you arrived in country less than one year before census date.

Reply:
Ah right, so the 'greater than' rule applies. Not confusing at all.

Reply:
it does, it was proved empirically in quick launcher.